packages/postcss-merge-rules/src/__tests__/index.js

```diff
@@ -576,3 +576,10 @@ test('should not crash when node.raws.value is null (2)', t => {
 
     t.deepEqual(res, '#foo .bar{ margin-left:auto; } #foo .bar,#foo .qux{ margin-right:auto; }');
 });
+
+test(
+    'should place rules with "all" at the top',
+    processCSS,
+    '.a{margin: 10px 20px;display: flex;all: initial;font-size: 10px;border-radius: 2px;color: blue;}.b{margin: 10px 30px;display: flex;all: initial;font-size: 20px;border-radius: 2px;color: blue;}',
+    '.a,.b{display: flex;all: initial;border-radius: 2px;color: blue;}.a{margin: 10px 20px;font-size: 10px;}.b{margin: 10px 30px;font-size: 20px;}'
```

Comment:
these are not equivalent. before minification "a" doesn't have margin because `all: initial` overrides it, after minification it does have margin, see https://codepen.io/vxsx/pen/jKONOZ

Reply:
you could argue that it's likely a mistake made by the css author, but css nano shouldn't fix that mistake for him

Reply:
@vxsx you are absolutely right. I guess the output should be like this:
```css
.a {
  margin: 10px 20px;
}

.b{
  margin: 10px 30px;
}

.a, .b{
  display: flex;
  all: initial;
  border-radius: 2px;
}

.a{
  font-size: 10px;
  color: blue;
}

.b{
  font-size: 20px;
  color: red;
}
```

This is still shorter than the original if class names are not too long.
What do you think?

Reply:
@chevsky can you send PR? :+1:

Reply:
<3

Reply:
This still appears to be broken. 

\cc @evilebottnawi

Reply:
/cc @chevsky friendly ping